### ISSUE TITLE
Training settings

### ISSUE THREAD

**Issue by leochli** [zhaolewen/DrQA-TF#3]
Hi lewen,

I am wondering how many epochs have you trained to reach the 56% EM accuracy? Besides, what's the F1 Score you finally got for the SQuAD dataset.
